web/server.js

```diff
@@ -261,6 +261,17 @@ module.exports = function(options) {
           reply(json.user);
         });
       }
+    },
+    {
+      method: 'POST',
+      path: '/logout',
+      config: {
+        auth: false
```

Comment:
If we do that can we still have it communicate the `state` back to the URL? In fact, if it communicates `state`, we don't even have to have a separate logout URL... we can just call the callback URL, and the client (i.e. teach site) can be responsible for setting state in such a way that it knows the callback was called in response to a logout, and any other necessary stuff it may need.